tutorials/cloud/aws/SageMaker_ASR_Training.ipynb

```diff
@@ -0,0 +1,418 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "2e86a2b3",
+   "metadata": {},
+   "source": [
+    "# NeMo ASR Training Using AWS SageMaker"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "215e3d3c",
+   "metadata": {},
+   "source": [
+    "In this tutorial we show how you can train a NeMo ASR Model using [Amazon SageMaker](https://docs.aws.amazon.com/sagemaker/latest/dg/whatis.html). This is meant to be a minimalistic example of how to use SageMaker with NeMo.\n",
+    "\n",
+    "AWS SageMaker is useful for practioners/researchers who are familiar with training locally or on a remote instance (via SSH). SageMaker also supports multi-GPU & Multi-node.\n",
+    "\n",
+    "Using AWS SageMaker we train a simple Conformer CTC model using the AN4 dataset on a remote instance with a GPU (p3.2xlarge). We use S3 to store the data and our checkpoints/logs.\n",
+    "\n",
+    "The overall steps are:\n",
+    "\n",
+    "1. Setup your AWS Credentials to access SageMaker\n",
+    "2. Download the source code we'll be running\n",
+    "3. Setup AN4 dataset, upload data to S3\n",
```

Comment:
Isnt AWS a paid resource ? AN4 for that purpose is a bit odd, but I suppose it would cost very little given the size of the dataset. Another option would be to showcase how to load a pretrained data from S3 or something, but that would be too expensive for a tutorial on the nemo side. I think AN4 is fine, just something to note for future consideration.

Reply:
In most cases, I assume that people would already have their data on S3 and just omit this honestly. That's fine, just showing someone how easy it is to bring their data to S3 if they haven't already (which I assume is unlikely)!